### ISSUE TITLE
Suggestion for changing the config from within zathura

### ISSUE THREAD

**Issue by ParadoxPacer** [gennaro-tedesco/zathuraconf#10]
Is there any way to change the color config from an already open zathura window using the input bar?

**Comment by gennaro-tedesco** [comment]
No, there isn't - this is on Zathura's: in order to change the configuration the application must reload the config file at start-up only.

**Comment by ParadoxPacer** [comment]
I actually found a way to keymap it with
`map <C-w> exec '/Users/[username]/go/bin/zathuraconf -f /Users/[username]/zathuraconf/themes/tokyonight.json'
map <C-c> exec '/Users/[username]/go/bin/zathuraconf -f /Users/[username]/zathuraconf/themes/catppuccin.json'`

You just need to point to the zathuraconf bin and give the absolute path to the .json files.
(BTW these examples apply specifically for a MacOS file system the location of the bin may be different on Linux)

and then what you need to do is enter ":source" in the inputbar and it will reload the page and change the colorscheme without having to close zathura

**Comment by gennaro-tedesco** [comment]
Ah interesting, I did not know, thank you for sharing!
